.ruby-gemset

```diff
@@ -0,0 +1 @@
+storycoder-sample_app
```

Comment:
gemsets are unique to your computer.  They will just have a common name.  This simply makes sure that you have a gemset